server/mvc/models/User.js

```diff
@@ -82,8 +73,8 @@ module.exports = function(){
     }
     return true;
   };
-  this.exists = function(){
-    return exists;
```

Comment:
`isSaved()` doesn't necessarily mean we called `save()`. It's also being set to `true` when `get()` finds a document in the database.